docs/resources/dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Workspace"
+---
+# databricks_dashboard Resource
+
+This resource allows you to manage Databricks [Dashboards](https://docs.databricks.com/en/dashboards/index.html). To manage [Dashboards](https://docs.databricks.com/en/dashboards/index.html) you must have a warehouse access on your databricks workspace.
+
+## Example Usage
+
+Dashboard using `serialized_dashboard` attribute:
+
+```hcl
+resource "databricks_dashboard" "dashboard" {
+			display_name			= 	"New Dashboard"
```

Comment:
we seem to have too many tabs here, could you run `make fmt-docs` as well

Reply:
I am unable to use terrafmt. but I manually formatted the docs

Reply:
you just need to install it as per: https://github.com/katbyte/terrafmt?tab=readme-ov-file#local-install